### ISSUE TITLE
Please add a LICENSE file

### ISSUE THREAD

**Issue by jaredsmith** [conventional-changelog/conventional-changelog#286]
Please add a LICENSE file -- The MIT license specifically states that the text of the license must accompany the source code.

**Comment by blaineadams** [comment]
I agree. In my environment we cannot use this package without a license file in the repository.

**Comment by bajtos** [comment]
FWIW, monorepo's root `package.json` says the license is ISC:

https://github.com/conventional-changelog/conventional-changelog/blob/35e60b5be6027fb2784c5103eee111f6f99b045e/package.json#L21

**Comment by hbetts** [comment]
Some packages are under the ISC license, while others are under the MIT license.

The root `package.json` is under the ISC, though that was setup automatically when this repository was converted to a mono-repo, and that particular license from that particular `package.json` file doesn't apply to any published package because the root `package.json` isn't published.

**Comment by hbetts** [comment]
My understanding is that ISC and MIT are functionally equivalent, so it may be worth looking into whether we can consolidate around a single license.

**Comment by bajtos** [comment]
> it may be worth looking into whether we can consolidate around a single license.

That would be great!

> Some packages are under the ISC license, while others are under the MIT license.

As far as I am concerned, having different licenses in different packages is fine. The issue I see is that some packages don't contain any `LICENSE` file, see e.g. [packages/conventional-commits-parser](https://github.com/conventional-changelog/conventional-changelog/tree/35e60b5be6027fb2784c5103eee111f6f99b045e/packages/conventional-commits-parser).

IMO, the first (and easy) step is to ensure every package has a LICENSE files matching the license entry in its `package.json` file.

**Comment by hbetts** [comment]
> IMO, the first (and easy) step is to ensure every package has a LICENSE files matching the license entry in its package.json file.

Agreed, along with the fact that it's just the right thing to do.

I think right now we're waiting on an update to #287

**Comment by hbetts** [comment]
I have not seen any progress on #287.

Would someone be available to submit another pull request containing a `LICENSE` file in each package within the `conventional-changelog` mono-repository?

**Comment by hbetts** [comment]
> I'll try to get to this today or tomorrow.

Thank you @jaredsmith! I've re-assigned the issue to you.

**Comment by jaredsmith** [comment]
Pull request #287 has been updated.

**Comment by hbetts** [comment]
Thanks to the work from @jaredsmith we have successfully published the following packages with new `LICENSE` files:
 - conventional-changelog-angular@3.0.4
 - conventional-changelog-atom@0.2.7
 - conventional-changelog-cli@1.3.19
 - conventional-changelog-codemirror@0.3.7
 - conventional-changelog-core@2.0.8
 - conventional-changelog-ember@0.3.9
 - conventional-changelog-eslint@1.0.8
 - conventional-changelog-express@0.3.6
 - conventional-changelog-jquery@1.3.7
 - conventional-changelog-jshint@0.3.7
 - conventional-changelog-preset-loader@1.1.8
 - conventional-changelog-writer@3.0.7
 - conventional-changelog@1.1.21
 - conventional-commits-parser@2.1.7
 - conventional-recommended-bump@2.0.8
 - git-raw-commits@1.3.6
 - git-semver-tags@1.3.6
 - gulp-conventional-changelog@1.1.21
 - standard-changelog@1.0.16

@bajtos and @blaineadams License issues should now be addressed. Please let us know if there are any other License related issues following these releases.
